CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/AnalysisScreenLoadingIndicatorAdapter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AnalysisScreenLoadingIndicatorAdapter.swift
+//  
+//
+//  Created by David Vizaknai on 12.09.2022.
+//
+
+import Foundation
+/**
+*   Adapter for injecting a custom loading indicator for the analysis viewcontroller.
+*/
+public protocol AnalysisScreenLoadingIndicatorAdapter: InjectedViewAdapter {
+    /**
+     *  Called when the screen is loaded, can be also called from the AnalysisViewController's showAnimation() method.
```

Comment:
For me the intention of the documentation here is to inform clients about what should they do when implementing this method. The questions the documentation should answer is: "As a client what do I have to do in this method? When is this method called? Why do I have to implement it?"

We don't need to mention the `showAnimation()` method here. It is currently public, but we'll have to make it internal when we remove the Component API. Clients won't be able to interact directly with our screen view controllers anymore.

My change suggestion would be:
```suggestion
     *  Called when the screen is loaded. You should start the loading indicator animation in this method.
```